samila/genimage.py

```diff
@@ -123,15 +123,17 @@ def plot(
         ax.add_artist(ax.patch)
         self.fig = fig
 
-    def nft_storage(self, api_key):
+    def nft_storage(self, api_key, depth=1):
```

Comment:
Add `DEFAULT_DEPTH` parameter to `params.py`
* Also update `save_image` method:
```python3
def save_image(self, file_adr, depth=DEFAULT_DEPTH):
    ...
def nft_storage(self, api_key, depth=DEFAULT_DEPTH):
    ...
```